### PR TITLE
Create offer

### PR DESCRIPTION
**Pull request by aashrafh**
Integration with backend
Annotation 2020-09-03 021703

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ghost) [inline .vscode/settings.json:1]: add VScode settings to gitignore

Comment (aashrafh) [reply .vscode/settings.json:1]: Done

**Review by AbdullahMuhammed5 — DISMISSED** [review]
Great work Ahmed, You just need to resolve some conflicts.

**Review by mohamedsgap — DISMISSED** [review]
LGTM :+1:

**Review by ghost — DISMISSED** [review]

**Review by mohamedsgap — APPROVED** [review]

**Review by AbdullahMuhammed5 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**